app/models/post.rb

```diff
@@ -1,3 +1,20 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  attribute :title, :string
+  attribute :text, :text
+  attribute :comments_counter, :integer, default: 0
+  attribute :likes_counter, :integer, default: 0
+
+  after_save :update_user_posts_counter
```

Comment:
- [ ] Great job-making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) :100: